transport/grpc/codec.go

```diff
@@ -0,0 +1,29 @@
+package grpc
+
+import "fmt"
+
+// PassThroughCodec passes bytes to/from the wire without modification
+type PassThroughCodec struct{}
```

Comment:
hmmmmmmmm. I use it in the examples/grpc/client and examples/grpc/server in future diffs as a way of setting up example client/servers, should I make a copy for those purposes?